.gitignore

```diff
@@ -24,6 +24,7 @@ speed-measure-plugin*.json
 *.sublime-workspace
 
 # IDE - VSCode
+.vscode
```

Comment:
doesn't that make the next line redundant?

Reply:
bb2aaad

Reply:
why not ignore the folder itself?

Reply:
It's the same, you cannot add an empty folder into git, you would at least create a .keep file.